packages/stockflux-launcher/public/app.dev.json

```diff
@@ -2,7 +2,7 @@
   "devtools_port": 9091,
   "runtime": {
     "arguments": "--enable-aggressive-domstorage-flushing",
-    "version": "13.76.43.31"
+    "version": "12.69.43.22"
```

Comment:
I don't think we should commit this change until we've agreed to roll everything back to 12. I think it's advisable to use version 12 while developing because of the issues we're seeing with 13, for now though